src/System.Runtime/tests/System/DoubleTests.netcoreapp.cs

```diff
@@ -141,6 +141,408 @@ public static void Parse_Span_Invalid(string value, NumberStyles style, IFormatP
+            CheckOneDouble("9324754620109615e+212", 0x6F43AE60753AF6CAul);
+            CheckOneDouble("78459735791271921e+049", 0x4D9DCD0089C1314Eul);
+            CheckOneDouble("272104041512242479e+200", 0x6D13BBB4BF05F087ul);
+            CheckOneDouble("6802601037806061975e+198", 0x6CF3BBB4BF05F087ul);
+            CheckOneDouble("20505426358836677347e-221", 0x161012954B6AABBAul);
+            CheckOneDouble("836168422905420598437e-234", 0x13B20403A628A9CAul);
+            CheckOneDouble("4891559871276714924261e+222", 0x7286ECAF7694A3C7ul);
+
+            //// values from http://www.icir.org/vern/papers/testbase-report.pdf table 2 (greater than half ULP - round up)
+            CheckOneDouble("85e-037", 0x38A698CCDC60015Aul);
+            CheckOneDouble("623e+100", 0x554640A62F3A83DFul);
+            CheckOneDouble("3571e+263", 0x77462644C61D41AAul);
+            CheckOneDouble("81661e+153", 0x60B7CA8E3D68578Eul);
+            CheckOneDouble("920657e-023", 0x3C653A9985DBDE6Cul);
+            CheckOneDouble("4603285e-024", 0x3C553A9985DBDE6Cul);
+            CheckOneDouble("87575437e-309", 0x016E07320602056Cul);
+            CheckOneDouble("245540327e+122", 0x5B01B6231E18C5CBul);
+            CheckOneDouble("6138508175e+120", 0x5AE1B6231E18C5CBul);
+            CheckOneDouble("83356057653e+193", 0x6A4544E6DAEE2A18ul);
+            CheckOneDouble("619534293513e+124", 0x5C210C20303FE0F1ul);
+            CheckOneDouble("2335141086879e+218", 0x6FC340A1C932C1EEul);
+            CheckOneDouble("36167929443327e-159", 0x21BCE77C2B3328FCul);
+            CheckOneDouble("609610927149051e-255", 0x0E104273B18918B1ul);
+            CheckOneDouble("3743626360493413e-165", 0x20E8823A57ADBEF9ul);
+            CheckOneDouble("94080055902682397e-242", 0x11364981E39E66CAul);
+            CheckOneDouble("899810892172646163e+283", 0x7E6ADF51FA055E03ul);
+            CheckOneDouble("7120190517612959703e+120", 0x5CC3220DCD5899FDul);
+            CheckOneDouble("25188282901709339043e-252", 0x0FA4059AF3DB2A84ul);
+            CheckOneDouble("308984926168550152811e-052", 0x39640DE48676653Bul);
+            CheckOneDouble("6372891218502368041059e+064", 0x51C067047DBB38FEul);
+
+            // http://www.exploringbinary.com/incorrect-decimal-to-floating-point-conversion-in-sqlite/
+            CheckOneDouble("1e-23", 0x3B282DB34012B251ul);
+            CheckOneDouble("8.533e+68", 0x4E3FA69165A8EEA2ul);
+            CheckOneDouble("4.1006e-184", 0x19DBE0D1C7EA60C9ul);
+            CheckOneDouble("9.998e+307", 0x7FE1CC0A350CA87Bul);
+            CheckOneDouble("9.9538452227e-280", 0x0602117AE45CDE43ul);
+            CheckOneDouble("6.47660115e-260", 0x0A1FDD9E333BADADul);
+            CheckOneDouble("7.4e+47", 0x49E033D7ECA0ADEFul);
+            CheckOneDouble("5.92e+48", 0x4A1033D7ECA0ADEFul);
+            CheckOneDouble("7.35e+66", 0x4DD172B70EABABA9ul);
+            CheckOneDouble("8.32116e+55", 0x4B8B2628393E02CDul);
```

Comment:
Can these be done instead with a Theory?

Reply:
Moved what inputs I could to use Theory/InlineData